src/common/config.cpp

```diff
@@ -84,6 +88,14 @@ void load(const std::filesystem::path& path) {
             isDebugDump = toml::find_or<toml::boolean>(debug, "DebugDump", false);
         }
     }
+    if (data.contains("LLE")) {
+        auto generalResult = toml::expect<toml::value>(data.at("LLE"));
+        if (generalResult.is_ok()) {
+            auto general = generalResult.unwrap();
+
+            isLibc = toml::find_or<toml::boolean>(general, "libc", true);
```

Comment:
Προτείνω το default να είναι false για οσους δεν έχουν libc.prx dumped

Reply:
basika ola ta paixnidia exoun bundled libc.prx , opote auto den einai issue . o monos logos pou iparxei auto einai an gia kapoio logo thes na xrisimopoiiseis to HLE libc , p.x se demos an kai den brisko kapoio logo iparksis sto mellon